sycl/test/extensions/group-algorithm.cpp

```diff
@@ -76,13 +76,13 @@ int main() {
   std::iota(input.begin(), input.end(), 0);
   std::fill(output.begin(), output.end(), 0);
 
-  test<class KernelNamePlusV>(q, input, output, plus<>(), 0, GeZero());
-  test<class KernelNameMinimumV>(q, input, output, minimum<>(),
+  test<class KernelNamePlusV>(q, input, output, ONEAPI::plus<>(), 0, GeZero());
```

Comment:
The newly added function objects in `sycl::` should be covered by tests as well, the same goes for the llvm-test-suite tests.

Reply:
I changed using of ONEAPI:: to using of sycl:: aliases in sycl/test. In test_suite/SYCL/GroupAlgorithm tests use ONEAPI functions instead sycl2020 and I left ONEAPI aliases there.  Should I add more tests to that directory?